system/gameModes/Sidewinder/sizeAdjustment.lst

```diff
@@ -2,88 +2,88 @@
 # $Id$
 #
 SIZENAME:Fine		ABB:F	BONUS:ITEMCOST|TYPE=Ammunition,TYPE=Armor,TYPE=Shield,TYPE=Weapon|0.5
-SIZENAME:Fine		BONUS:ITEMWEIGHT|TYPE=Ammunition,TYPE=Armor,TYPE=Shield,TYPE=Weapon|0.1	BONUS:ITEMWEIGHT|TYPE=Goods|0.25
-SIZENAME:Fine		BONUS:ACVALUE|TYPE.Armor,TYPE.Shield|0.5	BONUS:COMBAT|AC|8|TYPE=Size	BONUS:COMBAT|TOHIT|8|TYPE=SIZE	BONUS:COMBAT|TOHIT.GRAPPLE|-24|TYPE=Size
-SIZENAME:Fine		BONUS:ITEMCAPACITY|TYPE=Goods|0.25
-SIZENAME:Fine		BONUS:SKILL|Hide|16|TYPE=SIZE
-SIZENAME:Fine		BONUS:LOADMULT|TYPE=SIZE|0.125|PRELEGSGTEQ:4
-#
+SIZENAME:F			BONUS:ITEMWEIGHT|TYPE=Ammunition,TYPE=Armor,TYPE=Shield,TYPE=Weapon|0.1	BONUS:ITEMWEIGHT|TYPE=Goods|0.25
```

Comment:
Actually, chatting with Tom, he'll add a DISPLAYNAME token in a new PR so that the SIZENAME:F can be used on all lines and there isn't a change in referencing fashion required.